src/params/travels.jl

```diff
@@ -0,0 +1,19 @@
+Base.@kwdef struct Holiday <: InfectionTravels
+    days::Float64 = 0
+    peak::Float64 = 0
+    height::Float64 = 0
+    minimum::Float64 = 0
+  end
+  
+  function (f::Holiday)(state::SimState, params::SimParams, infection_time::Real=0.0)
+    #@assert kind(event) == OutsideTransmissionEvent
+    days = (f.days/2)^2
+    rand(state.rng) < max((Float64(-abs(infection_time-f.peak)^2)*f.height)/days + f.height, f.minimum)
+  end
+  
+  # This all to avoid using @eval and others
+  const travels = Dict{String, Type{T} where T}(
+      "Holiday" => Holiday
```

Comment:
Please add another function to behave similarly as the current `outsidefeed!` to be able to also use that